pytket/docs/index.rst

```diff
@@ -16,8 +16,6 @@ Windows. To install, run
 
 If you have issues installing ``pytket`` please visit the `installation troubleshooting <https://cqcl.github.io/tket/pytket/api/install.html>`_ page.
 
-To use ``pytket``, you can simply import the appropriate modules into your python code or in an interactive Python notebook. We can build circuits directly using the ``pytket`` interface by creating a blank circuit and adding gates in the order we want to apply them.
```

Comment:
Why delete this?

Reply:
The text seemed a bit redundant to me. If preferred I can leave it for now and update this is a fresh PR.